src/ripple/protocol/impl/SField.cpp

```diff
@@ -286,6 +301,20 @@ CONSTRUCT_TYPED_SFIELD(sfNFTokenOffers,         "NFTokenOffers",        VECTOR25
 // path set
 CONSTRUCT_UNTYPED_SFIELD(sfPaths,               "Paths",                PATHSET,    1);
 
+// Issue
+CONSTRUCT_TYPED_SFIELD(sfLockingChainIssue,      "LockingChainIssue",   ISSUE, 1);
+CONSTRUCT_TYPED_SFIELD(sfIssuingChainIssue,      "IssuingChainIssue",   ISSUE, 2);
+
+// Bridge
+CONSTRUCT_TYPED_SFIELD(sfXChainBridge,           "XChainBridge",        XCHAIN_BRIDGE,
+                                                                               1);
+
+// Attestation batch
+CONSTRUCT_TYPED_SFIELD(sfXChainAttestationBatch,
+                                                 "XChainAttestationBatch",
+                                                                         XCHAIN_ATTESTATION_BATCH,
```

Comment:
From looking at TxFormats.cpp, it looks like this should simply be a `TxFormat`, possibly containing one of more `InnerObjectFormat`s.  Is there a good reason not to do that?

Reply:
Attestation batch was removed, so this is no longer relevant.